neofetch

```diff
@@ -30,7 +30,8 @@
 
 version=7.0.0
 
-bash_version=${BASH_VERSION/.*}
+bash_version=${BASH_VERSINFO[0]:-5}
+shopt -s eval_unsafe_arith &>/dev/null
```

Comment:
```sh
# Fallback to a value of '5' for shells which support bash
# but do not set the 'BASH_' shell variables (osh). 
bash_version=${BASH_VERSINFO[0]:-5}
shopt -s eval_unsafe_arith &>/dev/null

```

Reply:
Note that I actually set `BASH_VERSION` later on. I could use `shell+=$BASH_VERSION; shell=${shell/-*}` instead

Reply:
:+1:

Reply:
Nevermind, that doesn't make sense. At all.

Reply:
Okay, I think what I was going for was:

```
if [[ $BASH_VERSION ]]; then
 shell+=$("$SHELL" -c "printf %s \"\$BASH_VERSION\"")
else
  shell+=$BASH_VERSION
fi
shell=${shell/-*}
```

Reply:
Or even something like:

```
shell+=${BASH_VERSION:-$("SHELL" -c "printf %s \"\$BASH_VERSION\"")}
shell=${shell/-*}
```

But if you're fine with that, then cool